v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -138,6 +138,115 @@ You [can associate an EC2 instance with an IAM role](https://docs.aws.amazon.com
+
+    ~~~json
+    {
+    "Version": "2012-10-17",
+    "Statement": {
+        "Effect": "Allow",
+        "Action": "sts:AssumeRole",
+        "Resource": "arn:aws:iam::{account ID}:role/{role name}"
+        }
+    }
+    ~~~
+
+    The `Resource` here is the Amazon Resource Name (ARN) of the role you created in step 1. You can copy this from the role's **Summary** page.
+
+    The `sts:AssumeRole` permission allows the user to obtain a temporary set of security credentials that gives them access to an S3 bucket to which they would not have access with their user-based permissions.
+
+1. <a name="step-3-assume"></a> Return to your IAM role's **Summary** page, and click on the **Trust Relationships** tab. Add a trust policy into the role, which will define the users that can assume the role. 
+
+    The following trust policy is providing this user the privilege to assume the role:
+
+    ~~~json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::123456789123:user/{user}"
+                },
+                "Action": "sts:AssumeRole"
+            }
+        ]
+    }
+    ~~~
+    When creating a trust policy consider the following:
+    - In the trust policy you need to include the ARN of the user that you want to assume the role under `Principal`. You can also include the [`Condition` attribute](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition.html) to further define access to the Amazon S3 bucket. For example, this could limit the operation within a date range, to users with multi-factor authentication enabled, or to specific IP addresses.  
+    - If you set the `Principal` ARN to `root`, this will allow any IAM user in the account with the `AssumeRole` permission to access the Amazon S3 bucket as per the defined IAM role permissions.
+    - When the IAM user takes on the role to perform a bulk operation, they have the permissions contained in the role. That is, not the permissions specified in their user profile.
+
+1. Run the bulk operation. It is necessary to pass in the S3 bucket's URL with the IAM user's access key ID and secret access key, if using [specified authentication](#specified-authentication). Otherwise, use `AUTH=IMPLICIT` for [implicit authentication](#implicit-authentication). For assuming the role, pass the assumed role's ARN, which you can copy from the IAM role's summary page:
+
+    ~~~sql
+    BACKUP DATABASE movr INTO 's3://{bucket name}?AWS_ACCESS_KEY_ID={user key}&AWS_SECRET_ACCESS_KEY={user secret key}&ASSUME_ROLE=arn:aws:iam::{account ID}:role/{role name}' AS OF SYSTEM TIME '-10s';
```

Comment:
Optional: would it be helpful here to note that we don't specify either value for `AUTH` because we are using `SPECIFIED` here, which is the default so need not be explicitly specified? Unsure, but took me a moment to remember that that's why our `AUTH` example didn't contain the `AUTH` param. Up to you!